src/main/java/it/gov/pagopa/pu/debtpositions/service/BaseDebtPositionOperationService.java

```diff
@@ -80,13 +80,21 @@ public WorkflowCreatedDTO execute(DebtPositionDTO debtPositionDTO, List<Installm
     applyOperation(debtPositionDTO, installments2operate, accessToken, org);
 
     debtPositionProcessorService.updateAmounts(debtPositionDTO);
-    debtPositionService.saveDebtPosition(debtPositionDTO);
 
-    debtPositionHierarchyStatusAlignerService.alignHierarchyStatus(debtPositionDTO);
+    saveDebtPosition(debtPositionDTO);
+    alignHierarchyStatus(debtPositionDTO);
```

Comment:
let's do this in the same method (they will always be execute together if needed or excluded both if not)
```suggestion
    saveAndAlignHierarchyStatus(debtPositionDTO);
```